rpmio/rpmstrpool.cc

```diff
@@ -26,9 +26,9 @@ struct poolHashBucket_s {
 };
 
 struct poolHash_s {
-    int numBuckets;
+    size_t numBuckets;
```

Comment:
poolHashCreate() and poolHashResize() numBuckets argument type should also change accordingly.